airbyte-integrations/connectors/destination-milvus/destination_milvus/indexer.py

```diff
@@ -46,12 +45,23 @@ def _create_client(self):
             proc.join()
             raise Exception("Connection timed out, check your host and credentials")
 
+    def _create_client(self):
+        self._connect_with_timeout()
         # If the process exited within 5 seconds, it's safe to connect on the main process to execute the command
         self._connect()
 
+        if not utility.has_collection(self.config.collection):
+            pk = FieldSchema(name="pk", dtype=DataType.INT64, is_primary=True, auto_id=True)
+            vector = FieldSchema(name=self.config.vector_field, dtype=DataType.FLOAT_VECTOR, dim=self.embedder_dimensions)
+            schema = CollectionSchema(fields=[pk, vector], enable_dynamic_field=True)
+            collection = Collection(name=self.config.collection, schema=schema)
+            collection.create_index(
+                field_name=self.config.vector_field, index_params={"metric_type": "L2", "index_type": "IVF_FLAT", "params": {"nlist": 1024}}
```

Comment:
I think it could be helpful to have a comment here explaining the `index_params` inputs.